cmd/containerd/command/service_windows.go

```diff
@@ -43,7 +43,6 @@ var (
 	logFileFlag           string
 
 	kernel32     = windows.NewLazySystemDLL("kernel32.dll")
-	setStdHandle = kernel32.NewProc("SetStdHandle")
 	allocConsole = kernel32.NewProc("AllocConsole")
```

Comment:
Wondering if we should upstream this one to golang.org/x/sys ?

Reply:
The more the merrier